### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by 78fatemanazari**
## Description
In this project, you will use transactions to update and delete records. You will also use your knowledge of aggregate functions and `GROUP BY` to answer analytical questions.

Insert data:
Capture
Rollback:
rollback
Delete all records:
delete
Savepoint:
savepoint
Multiply to -1:
multiply
Positive:
positive
Queries:
First:
qq-1
Second:
qq2
Third:
qq3
Fourth:
qq4
Fifth - part one:
qq5
Fifth - part two:
qq5-2
Sixth:
qq6

**Make Required Changes:**

first
second
third
fourth

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -15,4 +16,43 @@ SELECT * From animals Where neutered = true ;
...
\ No newline at end of file
+SELECT * From animals Where (weight_kg >= '10.4' and weight_kg <= '17.3');
+
+/* Second Project*/
+
+--PART ONE - transactions - species column and insertion
+UPDATE animals set species = 'digimon' Where name Like '%mon';
+--PART TWO - transactions - species column and insertion
+UPDATE animals set species = 'pokemon' Where species is null;
+
```
Comment (DeliceLydia) [inline queries.sql]: - [ ] Please make sure you `verify that change was made` and `verify that change persists after commit`
```sql
BEGIN;
   UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon'; 
   UPDATE animals SET species = 'pokemon' WHERE species IS NULL; 

   SELECT species from animals; 
   COMMIT;
   SELECT species from animals; 
commit
```

File: queries.sql [hunk 2]
```diff
@@ -15,4 +16,43 @@ SELECT * From animals Where neutered = true ;
 /* The 7th qeury*/
 SELECT * From animals Where name not in ('Gabumon'); 
 /* The 8th query*/
-SELECT * From animals Where (weight_kg >= '10.4' and weight_kg <= '17.3');
\ No newline at end of file
+SELECT * From animals Where (weight_kg >= '10.4' and weight_kg <= '17.3');
+
+/* Second Project*/
```
Comment (DeliceLydia) [inline queries.sql:21]: - [ ] Please create a `query` that `UPDATE` the animal's table by setting the species column to unspecified, then roll back the change
```sql
 BEGIN; -- start transaction

   UPDATE animals
   SET species = 'unspecified'; -- make change

   SELECT species from animals; -- verify that change was made
   ROLLBACK;

   SELECT species from animals; `-- verify that change was undone`
```

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @78fatemanazari, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Added a new column called `species`
- Added data in the table
- Followed git workflow


## Required Changes ♻️
_Check the comments under the review._



### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by abdulhamiid — APPROVED** [review]
# :green_circle:  Approved! :partying_face:
Hi @78fatemanazari 👋, 
image
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🌟

- ✔️ Use database transactions.
- ✔️ Modify and delete data in SQL.

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**